react/src/screens/BookDetail/index.js

```diff
@@ -25,8 +31,8 @@ class BookDetail extends Component {
         </NavLink>
         <div className="book-detail">
           <BookDetailData {...this.props} />
-          <BookDetailSuggestions />
-          <BookDetailCommentaries  bookId={this.props.match.params.id}/>
+          <BookDetailSuggestions bookId={Number(this.props.match.params.id)} />
+          <BookDetailCommentaries bookId={Number(this.props.match.params.id)} />
```

Comment:
We could receive `id` instead of `match.params.id` by manipulating this props in `mapStateToProps`. What do you think ?